ui/app/templates/components/auth-form.hbs

```diff
@@ -55,7 +55,7 @@
       @roleName={{this.roleName}}
       @selectedAuthType={{this.selectedAuthBackend.type}}
       @selectedAuthPath={{or this.customPath this.selectedAuthBackend.id}}
-      @disabled={{authenticate.isRunning}}
+      @disabled={{or authenticate.isRunning this.isLoading}}
```

Comment:
Added this because once the popup is closed, while things are processing but before authenticate runs, the button is clickable again and it was confusing

Reply:
Smart